smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -84,6 +84,18 @@ class Program
         [Option("-h|--use-http=<hostname>", Description = "Modules talk to iotedged via tcp instead of unix domain socket")]
         public (bool useHttp, string hostname) UseHttp { get; } = (false, string.Empty);
 
+        [Option("--use-connect-management-uri=<connect_management_uri>", Description = "Modules talk to a custom connect management socket (default is unix:///var/run/iotedge/mgmt.sock)")]
+        public (bool useConnMan, string connect_management_uri) UseConnMan { get; } = (false, string.Empty);
```

Comment:
why do you need to return (bool, string)? return string should be good enough, right?

Reply:
Good suggestion. Simplified it.